[gtk3] Unable to build

```
checking for CAIRO_BACKEND... no
configure: error: Package requirements ( cairo-xlib cairo) were not met:

No package 'cairo-xlib' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables CAIRO_BACKEND_CFLAGS
and CAIRO_BACKEND_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
gtk3 failed to build: `./configure --prefix=/usr/local --libdir=/usr/local/lib` exited with 1
```

This is the result on `armv7l` after a fresh install of cairo:
```
  GISCAN   Gdk-3.0.gir
Couldn't find include 'Pango-1.0.gir' (search path: '['../gdk', u'gir-1.0', u'/usr/local/share/gir-1.0', u'/usr/share/gir-1.0', u'/usr/local/share/gir-1.0']')
make[4]: *** [/usr/local/share/gobject-introspection-1.0/Makefile.introspection:160: Gdk-3.0.gir] Error 1
make[4]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26/gdk'
make[3]: *** [Makefile:1652: all-recursive] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26/gdk'
make[2]: *** [Makefile:1089: all] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26/gdk'
make[1]: *** [Makefile:722: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26'
make: *** [Makefile:616: all] Error 2
gtk3 failed to build: `make V=0 -j4` exited with 2
```

`armv7l` works fine now but still having an issue with `x86_64`.  See below:
```
  CCLD     libgdk-3.la
libtool:   error: cannot find the library '/usr/local/lib/libfreetype.la' or unhandled argument '/usr/local/lib/libfreetype.la'
make[4]: *** [Makefile:1170: libgdk-3.la] Error 1
make[4]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26/gdk'
make[3]: *** [Makefile:1652: all-recursive] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26/gdk'
make[2]: *** [Makefile:1089: all] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26/gdk'
make[1]: *** [Makefile:722: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/gtk+-3.22.26.tar.xz.dir/gtk+-3.22.26'
make: *** [Makefile:616: all] Error 2
gtk3 failed to build: `make V=0 -j2` exited with 2
```

--- a/packages/anagram.rb
+++ b/packages/anagram.rb
@@ -8,8 +8,16 @@ class Anagram < Package
   source_sha256 'd046fd5accd3c62267c0ef81b56cd05c59ec92b37cdb73f69d031879dba308bd'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/anagram-1.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/anagram-1.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/anagram-1.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/anagram-1.4-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '5d5e2fa0868d99ed24623a1fd98bf9a00a07c50e5a8c931cf2d43ab3dfbf1a42',
+     armv7l: '5d5e2fa0868d99ed24623a1fd98bf9a00a07c50e5a8c931cf2d43ab3dfbf1a42',
+       i686: '33dbe4fd1116e429e1b927f63de8361b3ba888f24712ae692bee0ac152bc41b7',
+     x86_64: '2d162c9d06d59ee1dab520deccfed04e3c27f7a5584346dfdc2e267bc46d310c',
   })
 
   def self.build

--- a/packages/ant.rb
+++ b/packages/ant.rb
@@ -3,13 +3,21 @@ require 'package'
 class Ant < Package
   description 'Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other.'
   homepage 'http://ant.apache.org/'
-  version '1.10.1'
-  source_url 'http://apache.mirror.vexxhost.com/ant/binaries/apache-ant-1.10.1-bin.zip'
-  source_sha256 '0acf6f46a71985912f9c2c768795b97e5c26bc9a7a0b61d27af8287f8b96cd8e'
+  version '1.10.2'
+  source_url 'https://www.apache.org/dist/ant/binaries/apache-ant-1.10.2-bin.tar.xz'
+  source_sha256 '361c8ad2ed8341416e323e7c28af10a8297170a80fdffba294a5c2031527bb6c'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ant-1.10.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ant-1.10.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ant-1.10.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ant-1.10.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '8a90bd6448e3937ff4d5c98e83ade7e3eb3af7b9e48ef541d16796bf17165305',
+     armv7l: '8a90bd6448e3937ff4d5c98e83ade7e3eb3af7b9e48ef541d16796bf17165305',
+       i686: 'db844b8b98315d929a9a1fec83280544cee3c003930ad901a0cfa079c534ece5',
+     x86_64: '0b0ba2ea0120b892b8ecba242fbad5897cdc7f695ff433d8ef632145426e90b6',
   })
 
   depends_on 'jdk8'

--- a/packages/aws_shell.rb
+++ b/packages/aws_shell.rb
@@ -8,8 +8,16 @@ class Aws_shell < Package
   source_sha256 '6d3a333c13c6c4cd3b9f1316a0a0c53ce4807b673dba63882ab2f834c2ffb903'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/aws_shell-0.2.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/aws_shell-0.2.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/aws_shell-0.2.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/aws_shell-0.2.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'a487ad655f5b496a4aa496bfd8fcccddf3f2c484f2d55e9b64070d3e9ea93c3c',
+     armv7l: 'a487ad655f5b496a4aa496bfd8fcccddf3f2c484f2d55e9b64070d3e9ea93c3c',
+       i686: '89e641e7093bd9fb08ae6886de51ceb6bf8162ea0c61b4e783002a388059b847',
+     x86_64: 'c88f2778b54fd031d1f74f4698c8dfc9fbde806c09990beef9335d32d83a1e9d',
   })
 
   depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"

--- a/packages/bitpocket.rb
+++ b/packages/bitpocket.rb
@@ -8,8 +8,16 @@ class Bitpocket < Package
   source_sha256 'f3952374a1139465700f9122d7a929227be5cdeb681679cbe00bb93658adbd1f'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bitpocket-0.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bitpocket-0.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/bitpocket-0.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bitpocket-0.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '7b94a65b172e2c93a8a35bd6c6642d61023b0c91ab9f2ba9a8db7f7dca9c150e',
+     armv7l: '7b94a65b172e2c93a8a35bd6c6642d61023b0c91ab9f2ba9a8db7f7dca9c150e',
+       i686: '42fcb4adb063c62d0a5907739b420e9a75457d51df44b64aa6b1600d2e81b3bf',
+     x86_64: '65b995eb9ea17bbce09c47d786cb1c38afac27952f2795fbcc4208b971e9de4a',
   })
 
   def self.install

--- a/packages/bmon.rb
+++ b/packages/bmon.rb
@@ -8,8 +8,16 @@ class Bmon < Package
   source_sha256 '02fdc312b8ceeb5786b28bf905f54328f414040ff42f45c83007f24b76cc9f7a'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bmon-4.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bmon-4.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/bmon-4.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bmon-4.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '1152f984aae1b54b4162a34460a8b3ced0be7e3c8a0887ac02fbd9d283cbd5aa',
+     armv7l: '1152f984aae1b54b4162a34460a8b3ced0be7e3c8a0887ac02fbd9d283cbd5aa',
+       i686: '5507412038511d2f5315478d3f527e54be02b3b9d69381aa1fbdf5a36621a5e0',
+     x86_64: 'e12f42c3d028678a37bf083ca9884ab4ad453de414f57a63eedec43c08705146',
   })
 
   depends_on 'libconfuse'

--- a/packages/clmystery.rb
+++ b/packages/clmystery.rb
@@ -8,8 +8,16 @@ class Clmystery < Package
   source_sha256 '911bc77225bcacc8fc1a7b17b21fbc4ede371513c729b4b663989ab627604d33'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/clmystery-6dd30f-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/clmystery-6dd30f-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/clmystery-6dd30f-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/clmystery-6dd30f-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'f7384340f728ba38105163d317cd827d079dbbddff846a0101127b90ecfc701c',
+     armv7l: 'f7384340f728ba38105163d317cd827d079dbbddff846a0101127b90ecfc701c',
+       i686: '9e440af530929d9afc4d9b02f6fda3cae503b029a830ed48341bc04ae76134b1',
+     x86_64: '94797d72a05da4875e8d258cf36c88ce6a1f6943eb371c900fcc55e512c75102',
   })
 
   def self.install

--- a/packages/cryptsetup.rb
+++ b/packages/cryptsetup.rb
@@ -7,6 +7,19 @@ class Cryptsetup < Package
   source_url 'https://www.kernel.org/pub/linux/utils/cryptsetup/v2.0/cryptsetup-2.0.1.tar.xz'
   source_sha256 '41d188092c52e23d576af41cf0cfe0555d8f7efa21598d4c57c56ea1b6d9c975'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/cryptsetup-2.0.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/cryptsetup-2.0.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/cryptsetup-2.0.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/cryptsetup-2.0.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'cfe406001f6f010a655a9b2a55ff3be03e29d5e2ee79a757a46bba3809b4ba01',
+     armv7l: 'cfe406001f6f010a655a9b2a55ff3be03e29d5e2ee79a757a46bba3809b4ba01',
+       i686: '8cd48aa7e01d5e6854e78a888f97478c530c18ee450809c9cc7867801a6cf3fa',
+     x86_64: 'fd33e4276dad999973373c75c802a0561b32a2472cc02c99b6b7bca07eb61eff',
+  })
+
   depends_on 'util_linux'
   depends_on 'lvm2'
   depends_on 'jsonc'

--- a/packages/gptfdisk.rb
+++ b/packages/gptfdisk.rb
@@ -7,6 +7,19 @@ class Gptfdisk < Package
   source_url 'https://sourceforge.net/projects/gptfdisk/files/gptfdisk/1.0.3/gptfdisk-1.0.3.tar.gz'
   source_sha256 '89fd5aec35c409d610a36cb49c65b442058565ed84042f767bba614b8fc91b5c'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gptfdisk-1.0.3-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gptfdisk-1.0.3-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gptfdisk-1.0.3-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gptfdisk-1.0.3-1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'afd7cdcf168c6d6097ea341b30fb52d62b2a007cf086d42cb8515f267b66784a',
+     armv7l: 'afd7cdcf168c6d6097ea341b30fb52d62b2a007cf086d42cb8515f267b66784a',
+       i686: 'e5af90a92886b5f9e7236345940ec9bb365332f105d65b2678153bca33b5ba46',
+     x86_64: 'eb8dd86aff1b9610681d764eef74351ed687ed307c519ef646ae2c8448017946',
+  })
+
   depends_on 'util_linux'
   depends_on 'ncurses'
   depends_on 'popt'

--- a/packages/graphviz.rb
+++ b/packages/graphviz.rb
@@ -7,6 +7,19 @@ class Graphviz < Package
   source_url 'https://gitlab.com/graphviz/graphviz/repository/67cd2e5121379a38e0801cc05cce5033f8a2a609/archive.tar.bz2'
   source_sha256 '8dddc80b4194b17c1a5bc1cc7b5e001e9a3ec27272287dc16f278c736a29a9b9'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/graphviz-2.40.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/graphviz-2.40.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/graphviz-2.40.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/graphviz-2.40.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'd1620cf88f88aca31001a64a26c95d02d5ee3b7109b093ea824f381f54d71ed0',
+     armv7l: 'd1620cf88f88aca31001a64a26c95d02d5ee3b7109b093ea824f381f54d71ed0',
+       i686: 'cb31be3ee6e1d74917c87ebb7a2bb2290e2a1cf1d60ef817b06a5d569b3fdcd3',
+     x86_64: 'd27c0177f7c154aacf6c69cbe0ae85c9a36a0aa2946fe32d5f4c9fd39772ce5a',
+  })
+
   depends_on 'automake' => :build
   depends_on 'pango'
   depends_on 'flex'

--- a/packages/gtk3.rb
+++ b/packages/gtk3.rb
@@ -7,6 +7,19 @@ class Gtk3 < Package
   source_url 'https://ftp.gnome.org/pub/gnome/sources/gtk+/3.22/gtk+-3.22.26.tar.xz'
   source_sha256 '61eef0d320e541976e2dfe445729f12b5ade53050ee9de6184235cb60cd4b967'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.22.26-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.22.26-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.22.26-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk3-3.22.26-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '797be95ad482f46e7d032a39f08fe69a41526f973a8c3f8f5b5a736e39f4a54b',
+     armv7l: '797be95ad482f46e7d032a39f08fe69a41526f973a8c3f8f5b5a736e39f4a54b',
+       i686: '76f237689e829c88368c4f6ec65832b8431008ac26d726f44ed7cc40fc67277e',
+     x86_64: 'bbf2cb9a8a2660fb4474ca1be8d18424f7c6099397b025c4f4917b7ea7789f30',
+  })
+
   depends_on 'gdk_pixbuf'
   depends_on 'libepoxy'
   depends_on 'graphene'

--- a/packages/gtkvte.rb
+++ b/packages/gtkvte.rb
@@ -7,6 +7,19 @@ class Gtkvte < Package
   source_url 'https://ftp.gnome.org/pub/gnome/sources/vte/0.48/vte-0.48.3.tar.xz'
   source_sha256 'a3a9fb182740b392a45cd3f46fa61a985f68bb6b1817b52daec22034c46158c3'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gtkvte-0.48.3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gtkvte-0.48.3-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gtkvte-0.48.3-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gtkvte-0.48.3-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '93f9d430e87dc5cf21aec88e4b32126ae0e16d272ebb4f220c4da57bd1e5d053',
+     armv7l: '93f9d430e87dc5cf21aec88e4b32126ae0e16d272ebb4f220c4da57bd1e5d053',
+       i686: '1a9026f4f4e3049a1b6de9eeed0ed5c67b4a91201074bb469ca09571d282269c',
+     x86_64: '818b9ff560977cf9f2498c12174040ccb75cc67bbc92de227b1f57dbd0900ec2',
+  })
+
   depends_on 'gtk3'
   depends_on 'libxml2'
   depends_on 'pcre2'

--- a/packages/gwt.rb
+++ b/packages/gwt.rb
@@ -8,8 +8,16 @@ class Gwt < Package
   source_sha256 '970701dacc55170088f5eb327137cb4a7581ebb4734188dfcc2fad9941745d1b'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gwt-2.8.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gwt-2.8.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gwt-2.8.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gwt-2.8.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'abb5a99c68ece5bac1c17b336fc6178ddee1b870a4e3491af31c1695a9b9d70d',
+     armv7l: 'abb5a99c68ece5bac1c17b336fc6178ddee1b870a4e3491af31c1695a9b9d70d',
+       i686: '533e96845f196c95510cec486bf4cd4f173a88b298a104eb6f8f9713a24f665b',
+     x86_64: 'ec69121900cf3370ad358465158e76967e5998a006604e522b102836c754f92b',
   })
 
   depends_on 'ant'

--- a/packages/hicolor_icon_theme.rb
+++ b/packages/hicolor_icon_theme.rb
@@ -7,6 +7,19 @@ class Hicolor_icon_theme < Package
   source_url 'https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.17.tar.xz'
   source_sha256 '317484352271d18cbbcfac3868eab798d67fff1b8402e740baa6ff41d588a9d8'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/hicolor_icon_theme-0.17-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/hicolor_icon_theme-0.17-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/hicolor_icon_theme-0.17-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/hicolor_icon_theme-0.17-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '3341e27bb5e2ea8c55b573c32bf34ba4e88da50c6cc1d865eb1c2d91232d44e6',
+     armv7l: '3341e27bb5e2ea8c55b573c32bf34ba4e88da50c6cc1d865eb1c2d91232d44e6',
+       i686: '574db604ef7db3b401e56ce4387e2a488952fcfa308316cc7127f872ee87557c',
+     x86_64: 'f0640114582b89e78c5e884e801d512b3a0f484443fdc0157a6f4db289261bae',
+  })
+
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

--- a/packages/jdk8.rb
+++ b/packages/jdk8.rb
@@ -8,8 +8,16 @@ class Jdk8 < Package
   source_sha256 'e632ae353ccfb90957cbb6a5818ee063710a41e7b97b490ee58ca5f627d863f1'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/jdk8-8u162-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/jdk8-8u162-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/jdk8-8u162-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/jdk8-8u162-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'bcbf7202fb97074c4161455d6a921378b4ce43b3fd430546b92d83d60c65fcc2',
+     armv7l: 'bcbf7202fb97074c4161455d6a921378b4ce43b3fd430546b92d83d60c65fcc2',
+       i686: '81d015255c8f1bbe9c78b6b78cc75cfd81fd63777ad8e1b1f1151d7f37d4889b',
+     x86_64: 'eebda392cdcdfee0cefac114edbfa4a39f8d76e83eb380e51e9bbc4a202206f1',
   })
 
   def self.install

--- a/packages/libcaca.rb
+++ b/packages/libcaca.rb
@@ -8,8 +8,16 @@ class Libcaca < Package
   source_sha256 '128b467c4ed03264c187405172a4e83049342cc8cc2f655f53a2d0ee9d3772f4'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libcaca-0.99.beta19-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libcaca-0.99.beta19-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libcaca-0.99.beta19-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libcaca-0.99.beta19-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '43d441fc33c3193911555bcc802944e18c00c7a94bb5d2d55daf993bba2343fc',
+     armv7l: '43d441fc33c3193911555bcc802944e18c00c7a94bb5d2d55daf993bba2343fc',
+       i686: '8a2aee2e085788984c0cd99ec4142ad330e593458a86ad9d976b6b6c46c143fa',
+     x86_64: '7683ba9192da59df87cf73573221d40de361ea2698fafdfb22713d1ccc11ecd9',
   })
 
   depends_on 'ncurses'

--- a/packages/libconfuse.rb
+++ b/packages/libconfuse.rb
@@ -8,8 +8,16 @@ class Libconfuse < Package
   source_sha256 '23c63272baf2ef4e2cbbafad2cf57de7eb81f006ec347c00b954819824add25e'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libconfuse-3.2.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libconfuse-3.2.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libconfuse-3.2.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libconfuse-3.2.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'c2bb1d942fe0cfd6545de8165ac1664fea1c4aa13e6d4a5c0f008aa98bc4202e',
+     armv7l: 'c2bb1d942fe0cfd6545de8165ac1664fea1c4aa13e6d4a5c0f008aa98bc4202e',
+       i686: '43abce4bd584a9953d7598ccfb36180322170f26f74331fc4b237e6abbbec64c',
+     x86_64: '697f525ada438cb1662374dc2ec820e2dd0133b339244ced2b56da0f098036e5',
   })
 
   depends_on 'apriconv'

--- a/packages/libmnl.rb
+++ b/packages/libmnl.rb
@@ -8,8 +8,16 @@ class Libmnl < Package
   source_sha256 '171f89699f286a5854b72b91d06e8f8e3683064c5901fb09d954a9ab6f551f81'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libmnl-1.0.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libmnl-1.0.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libmnl-1.0.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libmnl-1.0.4-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'dc5154a2d9bfcb43967639fc79097831dfb2668895c8b61ae889b8277536de0a',
+     armv7l: 'dc5154a2d9bfcb43967639fc79097831dfb2668895c8b61ae889b8277536de0a',
+       i686: 'a40aabf83e469e6e8fe8fa3a36d644f44dd23d86a83338b1c040029569b830bd',
+     x86_64: 'a67ed03507bb290ce861336a7cfd7f1a45a16b25b4c1a9abef1262b6ede7f0b8',
   })
 
   def self.build

--- a/packages/libxft.rb
+++ b/packages/libxft.rb
@@ -7,6 +7,19 @@ class Libxft < Package
   source_url 'https://www.x.org/archive/individual/lib/libXft-2.3.2.tar.gz'
   source_sha256 '26cdddcc70b187833cbe9dc54df1864ba4c03a7175b2ca9276de9f05dce74507'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libxft-2.3.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libxft-2.3.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libxft-2.3.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libxft-2.3.2-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '0ec0c51fca352e295132a541f2972c8811b4d9c6277adce53e8e843af5356e6e',
+     armv7l: '0ec0c51fca352e295132a541f2972c8811b4d9c6277adce53e8e843af5356e6e',
+       i686: '7dfcb1711c0c654518f3da4921de49a09b90dfe9621baa2d20a1ca61645c5daf',
+     x86_64: 'bc523b75dd9dbd750809c7865a75f896b4a12ee9089a8de8a64edd162a723ac8',
+  })
+
   depends_on 'libxrender'
   depends_on 'libx11' 
   depends_on 'fontconfig'

--- a/packages/miniconda3.rb
+++ b/packages/miniconda3.rb
@@ -7,6 +7,19 @@ class Miniconda3 < Package
   source_url 'https://raw.githubusercontent.com/Anaconda-Platform/anaconda-project/adb2d443b805f2c6c53f989251cc1a2b13fc0d0e/README.md'
   source_sha256 'ec0bfe39423ca117ffcd17c154e3e5f6c81a28c4fb14c22dd5033f499a306362'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/miniconda3-5.0.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/miniconda3-5.0.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/miniconda3-5.0.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/miniconda3-5.0.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'f39e37069874e57911e58024e63acdbe530b8b4a7cbf3197b86fd7a9f4855bd9',
+     armv7l: 'f39e37069874e57911e58024e63acdbe530b8b4a7cbf3197b86fd7a9f4855bd9',
+       i686: '3dc928dc63f119c92029213a92b244543e916c1ba239483a935ccac79da1b7b6',
+     x86_64: 'f488aed6b483c82f42be176b07fd8398b302381d60c80a34f3e7569a2295812a',
+  })
+
   depends_on 'python3'
 
   def self.install

--- a/packages/percona_boost.rb
+++ b/packages/percona_boost.rb
@@ -8,8 +8,16 @@ class Percona_boost < Package
   source_sha256 '727a932322d94287b62abb1bd2d41723eec4356a7728909e38adb65ca25241ca'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/percona_boost-1.59.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/percona_boost-1.59.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/percona_boost-1.59.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/percona_boost-1.59.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'a8ae7d1682a5801169991b3d824402059ea7b527ca1b118dca240d9a91201fd1',
+     armv7l: 'a8ae7d1682a5801169991b3d824402059ea7b527ca1b118dca240d9a91201fd1',
+       i686: '92bf850fafa7e11eee4040faf615ac97f31e8e561016d4fcbfa97d9ef6931452',
+     x86_64: '4cfcf14440dae594c3ad0c8fa3086f08c0e90787785d82e1b54454bcf0941ed9',
   })
 
   depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"

--- a/packages/percona_server.rb
+++ b/packages/percona_server.rb
@@ -8,8 +8,16 @@ class Percona_server < Package
   source_sha256 '17f06c07165954e7eacc3ba2cac0e1d4ba65b0b568f2437298d3c17a6a0940fd'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/percona_server-5.7.20-19-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/percona_server-5.7.20-19-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/percona_server-5.7.20-19-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/percona_server-5.7.20-19-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: 'e7bdb895946e539145268d29a0c93e690ad1ff91eab871605cb2b3b686247ff5',
+     armv7l: 'e7bdb895946e539145268d29a0c93e690ad1ff91eab871605cb2b3b686247ff5',
+       i686: '757706998388bf559a70f53567af4518c523f1236a500e228210ce023ef41a5c',
+     x86_64: '47431d59582a5be3361cbfd571de3da9a557b76cea40a01f4b58347988e7aa2c',
   })
 
   depends_on 'cmake' => :build

--- a/packages/perl_text_wrapi18n.rb
+++ b/packages/perl_text_wrapi18n.rb
@@ -7,6 +7,19 @@ class Perl_text_wrapi18n < Package
   source_url 'https://cpan.metacpan.org/authors/id/K/KU/KUBOTA/Text-WrapI18N-0.06.tar.gz'
   source_sha256 '4bd29a17f0c2c792d12c1005b3c276f2ab0fae39c00859ae1741d7941846a488'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/perl_text_wrapi18n-0.06-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/perl_text_wrapi18n-0.06-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/perl_text_wrapi18n-0.06-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/perl_text_wrapi18n-0.06-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '3978d40b0eb71c01e5a26d5e4f9069f41a7a8a477db3f1e20060274f503a814a',
+     armv7l: '3978d40b0eb71c01e5a26d5e4f9069f41a7a8a477db3f1e20060274f503a814a',
+       i686: 'c258b217a57569491a4200c92e07688fc7333b57207025ec760199218ac6a242',
+     x86_64: '5725c4a81381834d1c3701e05413d766d6882f98e7a0daf793cadb93e9a20962',
+  })
+
   depends_on 'perl'
   depends_on 'perl_text_charwidth'  # add dependency
 

--- a/packages/sqlite.rb
+++ b/packages/sqlite.rb
@@ -8,8 +8,16 @@ class Sqlite < Package
   source_sha256 '2824ab1238b706bc66127320afbdffb096361130e23291f26928a027b885c612'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/sqlite-3.22.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/sqlite-3.22.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/sqlite-3.22.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/sqlite-3.22.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '5b7f9778a780ab585a278f2c5683fe0db797862bb38d1d5800dfbacfa1b013f6',
+     armv7l: '5b7f9778a780ab585a278f2c5683fe0db797862bb38d1d5800dfbacfa1b013f6',
+       i686: 'aa56673a95856c12a8062493e47648f69cb6e94461dd75ef26477535f39647ba',
+     x86_64: '94d9f6a054d92a09c04324c0a95d6af4d2cc155dfac2a72879479d7a5d6129b8',
   })
 
   depends_on 'libedit'

--- a/packages/umockdev.rb
+++ b/packages/umockdev.rb
@@ -8,6 +8,19 @@ class Umockdev < Package
   source_url 'https://github.com/martinpitt/umockdev/archive/0.9.4.tar.gz'
   source_sha256 '9f13e8a836ef36158e2cc2b679cb99639e1f68a430b04fbe4dd07d3cb2b32a75'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/umockdev-0.9.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/umockdev-0.9.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/umockdev-0.9.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/umockdev-0.9.4-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '62d6ad9b0cbcbd6c50634e7ee0e77365b810b3a0be57fb61fbde768018428e92',
+     armv7l: '62d6ad9b0cbcbd6c50634e7ee0e77365b810b3a0be57fb61fbde768018428e92',
+       i686: '69a3a9f89569b72bf0c61aaf46ee045d02ef5de23e8cca169a207c4ac89bfd80',
+     x86_64: '95fd0e8eeacb4d95051cf80a1d6a44e4882c51f4be477f100c1d69e43f61fbd4',
+  })
+
   depends_on 'automake' => :build
   depends_on 'libtool' => :build
   depends_on 'intltool' => :build

--- a/packages/vala.rb
+++ b/packages/vala.rb
@@ -7,6 +7,19 @@ class Vala < Package
   source_url 'https://download.gnome.org/sources/vala/0.39/vala-0.39.7.tar.xz'
   source_sha256 'a54ec96125fdb63dbf96a8a3a65df41fdd88a617c1354088b5e10f8f0c89914d'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/vala-0.39.7-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/vala-0.39.7-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/vala-0.39.7-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/vala-0.39.7-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '1d8eb6026b3f61f5ec7d53f9fd62128b1458e3f6a1f99a0945d02bb98704b87a',
+     armv7l: '1d8eb6026b3f61f5ec7d53f9fd62128b1458e3f6a1f99a0945d02bb98704b87a',
+       i686: 'ff2aec0b381ecd54bd5fb043b1c4d7bfd0810665dea570ba3b3ff9683cbfb2c9',
+     x86_64: '798c51833e5110b45003a1efd2f3b6cf60dfc26f9c3f7d7158b3a1185d7a8f82',
+  })
+
   depends_on 'flex'
   depends_on 'graphviz'
   depends_on 'libxslt'

